x/clp/genesis_test.go

```diff
@@ -71,6 +71,8 @@ func CreateState(ctx sdk.Context, keeper clp.Keeper, t *testing.T) (int, int) {
 	for _, lp := range lpList {
 		keeper.SetLiquidityProvider(ctx, lp)
 	}
+	V1 := test.GenerateValidatorAddress("A58856F0FD53BF058B4909A21AEC019107BA6")
```

Comment:
Can we extract A58856F0FD53BF058B4909A21AEC019107BA6 to a constant?  We use it in a lot of places.